### ISSUE TITLE
Link performance issue

### ISSUE THREAD

**Issue by aepyornis** [public-accountability/littlesis-rails#62]
from the profiler:
```
GET http://ls.dev:8080/entities/33277	221.2	+0.0	6 sql	3.6
  Executing action: show	3770.7	+212.0	1942 sql	893.7
   Rendering: entities/show	23.7	+3983.0	
```
with many calls like this:

```
app/models/entity.rb:147:in `extension_ids'
app/models/entity.rb:162:in `extension_names'
app/models/link.rb:33:in `is_pfc_link?'
app/utility/sorted_links.rb:80:in `block in create_subgroups'
app/utility/sorted_links.rb:80:in `each'
app/utility/sorted_links.rb:80:in `partition'
app/utility/sorted_links.rb:80:in `create_subgroups'
app/utility/sorted_links.rb:26:in `initialize'
app/controllers/entities_controller.rb:9:in `new'
app/controllers/entities_controller.rb:9:in `show'
SELECT `extension_record`.`definition_id` FROM `extension_record` WHERE `extension_record`.`entity_id` = 182805
```

**Comment by aepyornis** [comment]
this function is culprit:
``` rb
def is_pfc_link?
    related.extension_names.include? 'PoliticalFundraising'
  end
```

and no matter what i do it won't eager load it correctly...sigh....

**Comment by aepyornis** [comment]
this particular problem was solved by 8e28b10
